### PR TITLE
Base64Decode-Decrypt-Encrypt secret value from a given file

### PR DESCRIPTION
**Pull request by ashwathishiva**
This PR contains:
- a convenience method to base64decode-decrypt-base64encode secret values from a given file to be able to handoff to K8s
- A minor fix to an issue that was causing `ReadFile()` to panic when toggling a secret between true and false

### CODE REVIEW

File: pkg/qliksense/context_configs.go [hunk 1]
```diff
@@ -422,3 +428,70 @@ func validateInput(input string) (string, error) {
...
+			return nil, err
+		}
+
+		// base64 encode the values
+		base64EncodedSecret = b64.StdEncoding.EncodeToString(decryptedString)
+		resultMap[k] = base64EncodedSecret
+	}
+	api.LogDebugMessage("B64 encoded Map: %v\n", resultMap)
+
+	return resultMap, nil
```
Comment (ffoysal) [inline pkg/qliksense/context_configs.go]: Need to return the file content as string like
```
apiVersion: v1
kind: Secret
metadata:
  name: whatever
data:
  key1: base64_encoded_value
  key2: base64_encoded_value
```

File: pkg/qliksense/context_configs.go [hunk 2]
```diff
@@ -422,3 +428,80 @@ func validateInput(input string) (string, error) {
...
+		// base64 encode the values
+		base64EncodedSecret = b64.StdEncoding.EncodeToString(decryptedString)
+		// resultMap[k] = ([]byte)base64EncodedSecret
+		resultMap[k] = base64EncodedSecret
+
+	}
+	api.LogDebugMessage("B64 encoded Map: %v\n", resultMap)
+
+	// putting the above map back into the k8sSecret struct
+	k8sSecret1.StringData = resultMap
```
Comment (ffoysal) [inline pkg/qliksense/context_configs.go]: isnt it `k8sSecret1.Data` instead of `k8sSecret1.StringData`

Comment (ashwathishiva) [reply pkg/qliksense/context_configs.go]: fixed

File: pkg/qliksense/context_configs.go (hunk 1 above)
Comment (ashwathishiva) [reply pkg/qliksense/context_configs.go]: Right. This will be the result now.

**Review by andriy-bulynko — APPROVED** [review]

**Pull request merged**